### DISCUSSION TITLE
Plugin mode use

### DISCUSSION THREAD

**Discussion by justlofe** [Erkko68/stable_player_display#2] [Q&A] [answered]
I am writing a parser for Paper plugin to use Animated Java animations. But here I have a question, what values and how I need to change in the exported json from plugin-mode to make stable_player_display work. I have seen a Python script for this, but I am not proficient in Python and don't understand it at all.

Here i assigned example json exported from plugin mode.
[example.json](https://github.com/user-attachments/files/18123240/example.json)

**Comment by Erkko68** [comment, marked as answer]
Apologies for the delayed response; GitHub doesn’t notify me when a discussion is opened. From your pull request, it seems like you’ve already resolved the issue, but I’m happy to clarify if needed.

The shader works by targeting specific entities using Y-axis translation values set to 0, 1024, 2048, and so on. The Python script scans the relevant files and folders for the field in the data modify function and replaces it with the corresponding bone translation (e.g., the head bone is 0, the right arm bone is 1024). Since there is already a value in that field, we need to sum it with the new value to ensure the animation behaves correctly, as you seem to have figured out.

This is my first time considering plugin mode, so I haven’t had the chance to test it yet. However, if you’re interested, I can try creating a Python script to add support for the plugin mode.
